src/quantize.rs

```diff
@@ -15,9 +17,13 @@ pub fn ac_q(qindex: usize) -> i16 {
     }
 }
 
-pub fn quantize_in_place(qindex: usize, coeffs: &mut [i32]) {
-    coeffs[0] /= dc_q(qindex) as i32;
-    let ac_quant = ac_q(qindex) as i32;
+pub fn quantize_in_place(qindex: usize, coeffs: &mut [i32], tx_size: TxSize) {
+    let tx_scale = match tx_size {
+        TxSize::TX_32X32 => 2,
+        _ => 1
+    };
+    coeffs[0] /= (dc_q(qindex) / tx_scale) as i32;
```

Comment:
Well, seeing again, 
doing coeffs[0] *= tx_scale; 
coeffs[0] /= dc_q(qindex) as i32;
does not lose the precision of dc_q(qindex)?

Reply:
Yeah I'll change it to do that.